pkg/branding/env.go

```diff
@@ -0,0 +1,17 @@
+package branding
+
+import "github.com/stackrox/rox/pkg/env"
+
+const (
+	// ProductBrandingEnvName should hold RHACS_BRANDING or STACKROX_BRANDING
+	ProductBrandingEnvName = "ROX_PRODUCT_BRANDING"
+)
+
+var (
+	productBrandingSetting = env.RegisterSetting(ProductBrandingEnvName, env.WithDefault("RHACS_BRANDING"))
+)
+
+// GetProductBrandingEnvName returns the environment variable ROX_BRANDING_NAME value
+func GetProductBrandingEnvName() string {
```

Comment:
1. Docstring and function name don't match, that causes style checks fail in CI.
2. There's no `ROX_BRANDING_NAME` any more.
3. The `Name` suffix is really confusing here.

There is environment variable, it is called `ROX_PRODUCT_BRANDING` its value can equal to `RHACS_BRANDING` or to something else (`STACKROX_BRANDING`, but that doesn't matter because we don't have the _third_ choice).

The value of `ROX_PRODUCT_BRANDING` is used to choose currently **one** _thing_ - visible product name in the report, but can be used to choose **additional** _things_ such as short product name, contact email, copyright text, etc.

Let's please not confused visible product name and the value of the environment variable.
I'd suggest to simply name this function as `GetProductBrandingEnv()`.

Reply:
Okay, done.